src/util.ts

```diff
@@ -44,9 +44,14 @@ export async function mkdtemp(closure: (dir: string) => Promise<void>) {
 }
 
 export async function synthApp(command: string, outdir: string, stdout: boolean, metadata: boolean): Promise<SynthesizedApp> {
+  if (!await fs.pathExists(outdir)) {
```

Comment:
Putting this here means that this warning will **always** be printed because the outdir is expected to be created further down (in the `await shell` call).